infra/charts/feast-feature-server/templates/service.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: http
+      targetPort: {{ .Values.feast_mode }}
```

Comment:
that's true

Reply:
yes... deployment and services are set to reflect the helm install chosen name

Reply:
initially, @redhatHameed wanted to append `feast_mode` to the deployment/service names but i discouraged it because i was concerned about orphaned objects should the mode be changed on an existing installation. based on his tests though it appears helm handles this fine, so if he wants to add it back it should be fine. As long as its thoroughly tested.

Reply:
oh, if it's the helm release name, then there are no conflicts, my fault.